Jenkinsfile

```diff
@@ -24,6 +27,16 @@ node {
             writeFile file: 'build-id', text: env.BUILD_NUMBER
             archiveArtifacts artifacts: "git-commit-id, build-id, external/**/*, internal/**/*", fingerprint: false
         }
+        stage("Deploy internal") {
+            if (isMaster) {
+                ftpPublisher alwaysPublishFromMaster: false, continueOnError: false, failOnError: true, publishers: [
+                    [
+                        configName: 'internal_doc_site',
+                        transfers: [ [removePrefix: 'internal', sourceFiles: 'internal/**/*', excludes: ''] ]
+                    ]
+                ]
+            }
```

Comment:
It'd be cool to also deploy the prod site for CD

Reply:
Yeah, we can totally do that. Probably in a future PR though as this was just a thing I added in my free time :)